domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/PartAdmissionResponse.java

```diff
@@ -0,0 +1,135 @@
+package uk.gov.hmcts.cmc.domain.models;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.google.common.collect.ImmutableList;
+import org.apache.commons.lang3.builder.ReflectionToStringBuilder;
+import org.hibernate.validator.constraints.NotBlank;
+import uk.gov.hmcts.cmc.domain.models.legalrep.StatementOfTruth;
+import uk.gov.hmcts.cmc.domain.models.party.Party;
+import uk.gov.hmcts.cmc.domain.models.response.DefendantPaymentPlan;
+import uk.gov.hmcts.cmc.domain.models.response.EvidenceItem;
+import uk.gov.hmcts.cmc.domain.models.response.HowMuchOwed;
+import uk.gov.hmcts.cmc.domain.models.response.TimelineEvent;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+
+import static uk.gov.hmcts.cmc.domain.utils.ToStringStyle.ourStyle;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@JsonInclude(JsonInclude.Include.NON_ABSENT)
+public class PartAdmissionResponse extends Response {
+
+    public enum PartAdmissionType {
+        AMOUNT_TOO_HIGH,
+        PAID_WHAT_BELIEVED_WAS_OWED
+    }
+
+    @NotNull
+    private final PartAdmissionType partAdmissionType;
+
+    @Valid
+    @NotNull
+    private final ImmutableList<EvidenceItem> evidence;
+
+    @Valid
+    @NotNull
+    private final ImmutableList<TimelineEvent> timeline;
+
+    @Valid
+    @NotNull
+    private final HowMuchOwed howMuchOwed;
+
+    @Valid
+    private final DefendantPaymentPlan defendantPaymentPlan;
+
+    @NotBlank
+    @Size(max = 99000)
+    private final String impactOfDispute;
```

Comment:
This field is optional in the frontend so it should be optional here as well. Remove `@NotBlank` annotation.